bindings/python/test/test_mongocrypt.py

```diff
@@ -107,28 +107,31 @@ def test_mongocrypt(self):
         kms_providers = {
             'aws': {'accessKeyId': 'foo', 'secretAccessKey': 'foo'}}
         opts = MongoCryptOptions(kms_providers)
-        mc = MongoCrypt(opts)
+        mc = MongoCrypt(opts, MockCallback())
         mc.close()
         mc.close()
 
     def test_mongocrypt_validation(self):
         with self.assertRaisesRegex(
                 TypeError, 'options must be a MongoCryptOptions'):
-            MongoCrypt({})
+            MongoCrypt({}, None)
         with self.assertRaisesRegex(
                 TypeError, 'options must be a MongoCryptOptions'):
-            MongoCrypt(None)
+            MongoCrypt(None, None)
 
-        invalid_key_len_opts = MongoCryptOptions({'local': {'key': b'1'}})
+        options = MongoCryptOptions({'local': {'key': b'\x00' * 96}})
         with self.assertRaisesRegex(
-                MongoCryptError, "local key must be 96 bytes"):
-            MongoCrypt(invalid_key_len_opts)
```

Comment:
I think we lost some test coverage for `local key must be 96 bytes` here.

Reply:
`libmongocrypt` was not throwing the expected error after using the new hook which is why I removed this check. Why are we checking this anyway, shouldn't that work be delegated to `libmongocrypt`?

Reply:
I think this is being fixed in https://github.com/mongodb/libmongocrypt/pull/122. We might need to rebase once that change is merged.

Reply:
I rebased this branch on #122 and added back the check and the tests pass (other than on PyPy of course).